scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,363 @@
+{% extends "layout_bs4.html" %}
+{% from "utils.html" import comments_panel, activity_panel, pedigree_panel %}
+{% from "variants/utils.html" import compounds_table, modal_causative %}
+{% from "variant/utils.html" import rankscore_panel, overlapping_panel, genes_panel, transcripts_panel, proteins_panel,  pin_button, causative_button, igv_track_selection %}
```

Comment:
Looks like there is something weird with how the template is used. If you look in the code further down `causative_button` is missing, but still you get the button showing on the top right menu:

![image](https://user-images.githubusercontent.com/28093618/115680846-3c595580-a354-11eb-9469-a4026b8b17a6.png)

It makes me think that the cancer variant is still using the variant.html template, could you check?

Reply:
Yes it like this, from cancer_variantS view, you click on a cancer variant, but you still see tthe variant page. Tried to debug with flash messages:

![image](https://user-images.githubusercontent.com/28093618/115682081-67907480-a355-11eb-8fb3-1d1765707a6e.png)


This is what you see in the page:
![image](https://user-images.githubusercontent.com/28093618/115682135-7840ea80-a355-11eb-859c-d70ee4a5e107.png)

Reply:
I dont  have that issue on my end. When I debug using the code you suggested flak seems to be using the correct templates.

However I fixed two minor issues in the cancer-variant template.

Reply:
Ok then something weird should have happened to my branch, I'll check and get back to you!

Reply:
This is super weird. I've tried to uninstall scout, reinstall and cleaned the browser history, just in case. 

Moved the debug message to the 2 templates instead so it's more direct. I still get that I'm on the variant.html template when visiting a cancer variant. 🤔 

@dnil, how does it look like for you?

Reply:
AHHHHH I KNOW!

There are 2 links for each variant on the cancer variantS page, one on the RANK, one on the gene_transcript:HGVS-thing column.

![image](https://user-images.githubusercontent.com/28093618/115824077-b0086a80-a407-11eb-854a-bf93db816a1e.png)


If you click on the rank you go to the variant page, if you click on the HGVS you go to the cancer variant page. Since I'm used to click on the rank, same as for rare disease variants, I'm going to the wrong template. Perhaps we should remove that rank link in this PR. Continuing with the review now! :)

Reply:
It is going away in @mhkc's other PR for the variantS-page, so should be fine - but well discovered!

Reply:
Ah! Great job. I could push a change if you want but as Daniel said its going away in the PR he is reviewing.